properties/service_fabric_managed_common.props

```diff
@@ -9,7 +9,7 @@
     <WarningLevel>4</WarningLevel>
     <NoWarn></NoWarn>
     <!-- Don't append TargetFramework to output path. -->
```

Comment:
Could you remove the comment, since it's no longer accurate?
Also, could you check if the AppentTargetFrameworkToOutputPath is true by default? If it is, I think we should remove it instead of setting it explicitly. Over time things like property names change and we often end up carrying misleading and obsolete code.